### ISSUE TITLE
Changing user role via CLI

### ISSUE THREAD

**Issue by sfc-gh-pkommini** [Netflix/dispatch#863]
**Describe the bug**
Changing the user role via cli seems to cause the error below:

```
root@xxxxyyyzzz:/# dispatch user update --role Admin first.last@company.com
/usr/local/lib/python3.8/site-packages/jose/backends/cryptography_backend.py:18: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes, int_to_bytes
Traceback (most recent call last):
  File "/usr/local/bin/dispatch", line 8, in <module>
    sys.exit(entrypoint())
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 636, in entrypoint
    dispatch_cli()
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 188, in update_user
    user_service.update(user=user, user_in=UserUpdate(id=user.id, role=role), db_session=db_session)
  File "/usr/local/lib/python3.8/site-packages/dispatch/auth/service.py", line 63, in update
    password = bytes(user_in.password, "utf-8")
TypeError: encoding without a string argument

```
**To Reproduce**
1. Go to web container
2. Type `dispatch user update --role Admin first.last@company.com`

**Expected behavior**
Update the user `first.last@company.com` to Admin

**Comment by kevgliss** [comment]
I believe this was fixed in #858

**Comment by sfc-gh-pkommini** [comment]
It works now. Thank you!
